### PR TITLE
fix: smaller then instead of bigger then :-( dumb mistake

### PR DESCRIPTION
**Pull request by RJPvT**
made a mistake last time. of course pending review is bigger when it is in the future instead of smaller

### CODE REVIEW

File: erpnext/projects/doctype/task/task.py [hunk 1]
```diff
@@ -202,7 +202,7 @@ def set_tasks_as_overdue():
 	tasks = frappe.get_all("Task", filters={'status':['not in',['Cancelled', 'Closed']]})
```
Comment (hrwX) [inline erpnext/projects/doctype/task/task.py]:
```suggestion
	tasks = frappe.get_all("Task", filters={'status':['not in',['Cancelled', 'Closed']]}, fields=["name", "review_date"])
```
It doesn't make sense to `frappe.db.get_value` when you can get all the values in `frappe.get_all`

**Review by hrwX — CHANGES_REQUESTED** [review]

File: erpnext/projects/doctype/task/task.py (hunk 1 above)
Comment (RJPvT) [reply erpnext/projects/doctype/task/task.py]: I could make this change but review date is not nessecary for most. In a test just now a standard query with this change is 1.2ms slower then the current setup. Should i make the change ?

File: erpnext/projects/doctype/task/task.py [hunk 2]
```diff
@@ -202,7 +202,7 @@ def set_tasks_as_overdue():
...
 	for task in tasks:
 		if frappe.db.get_value("Task", task.name, "status") in 'Pending Review':
-			if getdate(frappe.db.get_value("Task", task.name, "review_date")) < getdate(today()):
+			if getdate(frappe.db.get_value("Task", task.name, "review_date")) > getdate(today()):
```
Comment (hrwX) [inline erpnext/projects/doctype/task/task.py]: If you get all the fields in `get_all`, it'll save a lot of `db.get_values` here, so its a no brainer change to make.

**Review by hrwX — CHANGES_REQUESTED** [review]

File: erpnext/projects/doctype/task/task.py (hunk 2 above)
Comment (RJPvT) [reply erpnext/projects/doctype/task/task.py]: Done

File: erpnext/projects/doctype/task/task.py [hunk 3]
```diff
@@ -199,10 +199,10 @@ def set_multiple_status(names, status):
 		task.save()
 
 def set_tasks_as_overdue():
-	tasks = frappe.get_all("Task", filters={'status':['not in',['Cancelled', 'Closed']]})
+	tasks = frappe.get_all("Task", filters={'status':['not in',['Cancelled', 'Closed']]}, fields=["name","status","review_date"])
```
Comment (hrwX) [inline erpnext/projects/doctype/task/task.py]:
```suggestion
	tasks = frappe.get_all("Task", filters={"status": ["not in", ["Cancelled", "Closed"]]}, fields=["name", "status", "review_date"])
```

Comment (hrwX) [inline erpnext/projects/doctype/task/task.py]:
```suggestion
	tasks = frappe.get_all("Task", filters={"status": ["not in", ["Cancelled", "Closed"]]}, fields=["name", "status", "review_date"])
```

File: erpnext/projects/doctype/task/task.py [hunk 4]
```diff
@@ -199,10 +199,10 @@ def set_multiple_status(names, status):
...
 	for task in tasks:
-		if frappe.db.get_value("Task", task.name, "status") in 'Pending Review':
-			if getdate(frappe.db.get_value("Task", task.name, "review_date")) > getdate(today()):
+		if task.status in 'Pending Review':
```
Comment (hrwX) [inline erpnext/projects/doctype/task/task.py]:
```suggestion
		if task.status == "Pending Review":
```

**Review by hrwX — CHANGES_REQUESTED** [review]

**Comment by hrwX** [comment]
@RJPvT Can you open the same PR for `version-12-hotfix` as well as `develop`

**Comment by RJPvT** [comment]
Yes, i will do that tonight.

**Comment by RJPvT** [comment]
Also changed Closed to Completed in v12 en develop. This was introduced in v12 en was apparently not working anyway :-)

**Comment by RJPvT** [comment]
@hrwX Are the PR's now up to code for implementation ? I have to manually reset 50+ tasks because of this.

**Review by nabinhait — APPROVED** [review]

**Pull request merged**